ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -217,18 +217,24 @@ extension KeymanWebViewController {
 
   func setText(_ text: String?) {
     var text = text ?? ""
+
     // Remove any system-added LTR/RTL marks.
     text = text.replacingOccurrences(of: "\u{200e}", with: "") // Unicode's LTR codepoint
     text = text.replacingOccurrences(of: "\u{200f}", with: "") // Unicode's RTL codepoint (v1)
     text = text.replacingOccurrences(of: "\u{202e}", with: "") // Unicode's RTL codepoint (v2)
```

Comment:
Does this mean our keyboards can never emit these? probably a separate issue because it may be needed in some circumstances.

Reply:
Pretty sure that's a yes.

Note that this is enforced because iOS itself will randomly add those characters in during cut/paste ops (at a minimum) and possibly in other contexts.  Failing to filter those out when passing 'em to KMW has caused major issues with `nul` rules in the past; I'm fairly certain a `git blame` could point to the PRs where these lines were added.